ui/app/components/clients/date-range.ts

```diff
@@ -62,6 +68,24 @@ export default class ClientsDateRangeComponent extends Component<Args> {
     return parseAPITimestamp(isoTimestamp, 'MMMM yyyy');
   };
 
+  get historicalBillingPeriods() {
+    const count = this.args.retentionMonths / 12;
```

Comment:
We should probably account for this not being divisible evenly by 12. Since we only want users to query full billing periods, we should probably wrap this calculation in `Math.floor()` (instead of `Math.ceil()`). But might be worth conferring with the backend.